src/displayBackend/wayland/PresentationFeedbackSurface.hpp

```diff
@@ -0,0 +1,63 @@
+/*
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, write to the Free Software
+ *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307 USA
+ *
+ * Copyright (C) 2022 EPAM Systems Inc.
+ *
+ */
+
+#pragma once
+
+#include "Surface.hpp"
+#include "Registry.hpp"
+
+#include "presentation-time-client-protocol.h"
+
+
+namespace Wayland {
+
+class PresentationFeedbackSurface : public Surface {
+public:
```

Comment:
Why do you need two public sections?
Add oxygen comment for all public methods.